tests/test_modeling_tf_longformer.py

```diff
@@ -548,3 +548,25 @@ def test_inference_masked_lm_long(self):
         tf.debugging.assert_near(tf.reduce_mean(loss), expected_loss, rtol=1e-4)
         tf.debugging.assert_near(tf.reduce_sum(prediction_scores), expected_prediction_scores_sum, rtol=1e-4)
         tf.debugging.assert_near(tf.reduce_mean(prediction_scores), expected_prediction_scores_mean, rtol=1e-4)
+
+    @slow
+    def test_inference_masked_lm(self):
+        model = TFLongformerForMaskedLM.from_pretrained("lysandre/tiny-longformer-random", use_cdn=False)
```

Comment:
Will remove the `use_cdn=False` tomorrow once the CDN caching has caught up.

```suggestion
        model = TFLongformerForMaskedLM.from_pretrained("lysandre/tiny-longformer-random")
```